util/test/account_testutil.go

```diff
@@ -602,3 +620,11 @@ func MakeGenesisBlock() bookkeeping.Block {
 	}
 	return genesisBlock
 }
+
+// ArbitraryString should be used to generate a pseudo-random string to put in the Note field of a Txn Header.
+// This is necessary to ensure the hash of any two txns used in tests are never the same.
+func ArbitraryString() []byte {
```

Comment:
Does this need to be public?

Reply:
Not within this PR since the only thing using it is in the same module, but I see no reason why it couldn't be used in another test module in the future. 

If you want, I can change it to private for this commit and if we need it elsewhere we can change it to public at that time?